public/sass/elements/_icons.scss

```diff
@@ -1,244 +1,102 @@
+// GOV.UK front end toolkit icons
+.icon-calendar {
+  @include icon(icon-calendar, 27, 27);
 }
 
-.icon-step-2 {
-  width: 23px;
-  height: 23px;
-  background-image: file-url("icons/icon-steps/icon-step-2.png");
-
-  @include device-pixel-ratio() {
-    background-image: file-url("icons/icon-steps/icon-step-2-2x.png");
-    background-size: 100%;
-  }
+.icon-file-download {
+  @include icon(icon-file-download, 30, 39);
 }
 
-.icon-step-3 {
-  width: 23px;
-  height: 23px;
-  background-image: file-url("icons/icon-steps/icon-step-3.png");
-
-  @include device-pixel-ratio() {
-    background-image: file-url("icons/icon-steps/icon-step-3-2x.png");
-    background-size: 100%;
-  }
+.icon-important {
+  @include icon(icon-important, 35, 35);
 }
 
-.icon-step-4 {
-  width: 23px;
-  height: 23px;
-  background-image: file-url("icons/icon-steps/icon-step-4.png");
-
-  @include device-pixel-ratio() {
-    background-image: file-url("icons/icon-steps/icon-step-4-2x.png");
-    background-size: 100%;
-  }
+.icon-information {
+  @include icon(icon-information, 27, 27);
 }
 
-.icon-step-5 {
-  width: 23px;
-  height: 23px;
-  background-image: file-url("icons/icon-steps/icon-step-5.png");
-
-  @include device-pixel-ratio() {
-    background-image: file-url("icons/icon-steps/icon-step-5-2x.png");
-    background-size: 100%;
-  }
+.icon-locator {
+  @include icon(icon-locator, 26, 36);
 }
 
-.icon-step-6 {
-  width: 23px;
-  height: 23px;
-  background-image: file-url("icons/icon-steps/icon-step-6.png");
-
-  @include device-pixel-ratio() {
-    background-image: file-url("icons/icon-steps/icon-step-6-2x.png");
-    background-size: 100%;
-  }
+.icon-pointer {
+  @include icon(icon-pointer, 30, 19);
 }
 
-.icon-step-7 {
-  width: 23px;
-  height: 23px;
-  background-image: file-url("icons/icon-steps/icon-step-7.png");
-
-  @include device-pixel-ratio() {
-    background-image: file-url("icons/icon-steps/icon-step-7-2x.png");
-    background-size: 100%;
-  }
+.icon-pointer-black {
+  @include icon(icon-pointer-black, 23, 23);
 }
 
-.icon-step-8 {
-  width: 23px;
-  height: 23px;
-  background-image: file-url("icons/icon-steps/icon-step-8.png");
-
-  @include device-pixel-ratio() {
-    background-image: file-url("icons/icon-steps/icon-step-8-2x.png");
-    background-size: 100%;
-  }
+.icon-search {
+  @include icon(icon-search, 30, 22);
 }
 
-.icon-step-9 {
-  width: 23px;
-  height: 23px;
-  background-image: file-url("icons/icon-steps/icon-step-9.png");
 
-  @include device-pixel-ratio() {
-    background-image: file-url("icons/icon-steps/icon-step-9-2x.png");
-    background-size: 100%;
+// GOV.UK step icons
+@for $i from 1 through 14 {
+  .icon-step-#{$i} {
+    @include icon(icon-step-#{$i}, 23, 23, icon-steps);
   }
 }
 
-.icon-step-10 {
-  width: 23px;
-  height: 23px;
-  background-image: file-url("icons/icon-steps/icon-step-10.png");
+// Propose replacing step icons with circles
+.circle {
+  display: inline-block;
+  @include border-radius(100%);
```

Comment:
Can we make this 50%, just in case future specs define something different when border radiuses butt up against each other?